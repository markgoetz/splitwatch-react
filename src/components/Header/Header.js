import React from 'react';
import Heading from '../Heading/Heading';
import styles from './Header.module.css';

export default function Header(props) {
  return (<div className={styles.headerBg}>
    <Heading level="1">SplitWatch</Heading>
  </div>);
}
