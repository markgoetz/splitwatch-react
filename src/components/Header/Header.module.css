.headerBg {
    color: var(--primary-color);
    background-color: var(--darkest-grey);
    text-align: center;
    padding: 20px 0;
    margin-bottom: 40px;
}