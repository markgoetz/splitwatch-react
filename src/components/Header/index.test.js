import React from 'react';
import Header from './Header';
import { shallow } from 'preact-render-spy';

describe('Header Component', () => {

  test('Header renders correctly', () => {
    const context = shallow(<Header />);
    expect(context).toMatchSnapshot();
  });
});