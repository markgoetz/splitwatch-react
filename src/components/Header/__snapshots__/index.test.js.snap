// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component Header renders correctly 1`] = `
preact-render-spy (1 nodes)
-------
<div>
  <Heading level="1">SplitWatch</Heading>
</div>

`;
